Pinta/Dialogs/ResizeCanvasDialog.cs

```diff
@@ -373,6 +372,8 @@ private void Build()
 			grid_align.Add (grid);
 
 			main_vbox.PackStart (grid_align, false, false, 0);
+			
+			main_vbox.PackStart (hbox_percent, false, false, 0);
```

Comment:
To me the layout looks a bit odd having the percentage option under the anchor buttons. Maybe just leave the layout as-is?